Lib/http/client.py

```diff
@@ -825,9 +825,10 @@ def _get_content_length(body, method):
         return None
 
     def __init__(self, host, port=None, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
-                 source_address=None):
+                 source_address=None, blocksize=8192):
         self.timeout = timeout
         self.source_address = source_address
+        self.blocksize = blocksize
```

Comment:
I suggest either name the attribute with an underscore (“_blocksize”) or document it as a public attribute.

Reply:
I'll document that this is a public attribute, since a user of the connection may like to modify it.

For example, user code may want to check if a connection supports changing the block size, and hasattr(conn, "blocksize") is nicer to use compared with checking __init__ signature.